azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -137,9 +138,9 @@
    * @throws ExecutorManagerException
    */
   public FlowRunner(ExecutableFlow flow, ExecutorLoader executorLoader,
-      ProjectLoader projectLoader, JobTypeManager jobtypeManager)
+      ProjectLoader projectLoader, JobTypeManager jobtypeManager, Props serverProps)
```

Comment:
Quick question: What is the differences between server props and azkaban props? I see different usages from the code. Does it make sense to combine the two ?